base/applications/network/netstat/netstat.c

```diff
@@ -536,7 +536,7 @@ VOID ShowUdpTable(VOID)
             PID[0] = 0;
         }
 
-        ConPrintf(StdOut, L"  %-6s %-22s %-34s %s\n", L"UDP", Host, L"*:*", PID);
+        ConPrintf(StdOut, L"  %-6s %-22S %-34S %S\n", L"UDP", Host, L"*:*", PID);
```

Comment:
This isn't right, the `L"*:*"` is a wide string, so the third `%s` should be lowercase